ThirdPartyAdapters/unity/unity/build.gradle

```diff
@@ -17,7 +17,7 @@ ext {
 android {
     compileSdkVersion 28
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 19
```

Comment:
Not sure if this will be a problem but I had to do this after I replaced unity_ads.aar

Reply:
this is fine, it should match our min sdk version for Unity Ads